### ISSUE TITLE
`farmer config` fails

### ISSUE THREAD

**Issue by defrex** [vmfarms/farmer#5]
After installing farmer using Arch Linux's system `pip` (as root), I receive the following error:
```
$ farmer config
Traceback (most recent call last):
  File "/usr/bin/farmer", line 11, in <module>
    sys.exit(cli())
  File "/usr/lib/python3.6/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1037, in invoke
    return Command.invoke(self, ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/farmer/commands/config.py", line 67, in cli
    if not config.get(config, 'token'):
  File "/usr/lib/python3.6/site-packages/layeredconfig/layeredconfig.py", line 151, in get
    if hasattr(config, key):
  File "/usr/lib/python3.6/site-packages/layeredconfig/layeredconfig.py", line 259, in __getattr__
    if source.has(name):
  File "/usr/lib/python3.6/site-packages/layeredconfig/dictsource.py", line 41, in has
    return key in self.source and not isinstance(self.source[key], type)
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by defrex** [comment]
I'm guessing this has to do with installing as root and running in userspace. That will be a common setup for non-pythong project.

**Comment by defrex** [comment]
Nope, I'm wrong. I get the same issue in a virtualenv.

**Comment by benvmfarms** [comment]
Thanks for the bug report, Aron. I can reproduce it if `~/.farmer/farmer.yml` is empty.

I believe I fixed this in `296f265`. We'll release a bugfix version once you confirm.

**Comment by defrex** [comment]
@benwebber confirmed
```
╔══ defrex@defrex-x1 venv ~/code/careguide/usn-dev/api:customerio!1 ════════════════════════════════════════════════════════════════╡10:15:49
╚╡echo 'key: value' > ~/.farmer/farmer.yml
╔══ defrex@defrex-x1 venv ~/code/careguide/usn-dev/api:customerio!1 ════════════════════════════════════════════════════════════════╡10:16:49
╚╡farmer config
Hi! It looks like you haven't configured Farmer yet.

Enter your VM Farms API token
```

**Comment by benvmfarms** [comment]
Brilliant, thanks! I've released 2.1.1 on PyPI.
